Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -328,11 +328,10 @@
 "reading-list-login-title" = "Sync your saved articles?";
 "reading-list-new-list-description-placeholder" = "optional short description";
 "reading-list-new-list-name-placeholder" = "reading list title";
-"reading-list-show-default-list-setting" = "Show default list";
 "reading-list-sync-enable-button-title" = "Enable syncing";
 "reading-list-sync-enable-subtitle" = "Your saved articles and reading lists can now be saved to your Wikipedia account and synced across devices.";
 "reading-list-sync-enable-title" = "Turn on reading list syncing?";
-"reading-list-sync-setting" = "Reading list sync";
+"reading-list-sync-setting" = "Saved articles storage and syncing";
```

Comment:
Looks like this uses same string as `settings-storage-and-syncing-title`. Consider consolidating these and exposing the shared string via CommonString if it seems like they're most likely going to remain the same.